src/common/deleteItem.js

```diff
@@ -0,0 +1,10 @@
+const deleteItem = (e) => {
+  e.stopPropagation();
+  const index = e.currentTarget.parentNode.childNodes[0].innerHTML;
+  const storedList = JSON.parse(localStorage.getItem('toDolist'));
+  storedList.splice(index, 1);
+  localStorage.setItem('toDolist', JSON.stringify(storedList));
+  window.location.reload();
+};
+
+export default deleteItem;
```

Comment:
@saikatgpae, I have noticed you have an empty file below this named: getData.js,  I would suggest removing this file from your project to keep your project clean.

Reply:
Thank you for your suggestion.
i removed the same.